about.html

```diff
@@ -0,0 +1,119 @@
+  gtag('js', new Date());
+
+  gtag('config', 'G-9S3Y96W739');
+</script>
+    <title>ABOUT HIR AFRICA</title>
+</head>
+<body>
+    <header class="header">
+        <div class="topNav">
+        <ul>
+        <li><a href="#"><i class="fab fa-facebook-f"></i></a></li>
+        <li><a href="#"><i class="fab fa-twitter"></i></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+        </ul>
+        </div>
+        <nav class="mainNav">
+          <ul class="mainNavList">
+            <li><a href="index.html"><img src="Images/Hirlogo1.jpg" alt="toplogo" class="mainLogo"></a></li>
+            <li><a href="index.html">Home</a></li>
+            <li><a href="index.html">Program</a></li>
+            <li><a href="#join">Join</a></li>
+            <li><a href="#sponsor">Sponsor</a></li>
+            <li><a href="#News">News</a></li>
+            <li><a href="#HirCompaign" class="hirCompaign">HIR Compaign</a></li>
+          </ul> 
+        </nav>
+    
+    <div><a href="index.html"><img src="Images/Hirlogo1.jpg" alt="logo" class="navLogo"></a></div>
+
+    <div class="mobileMenuIcon"><i class="fas fa-bars bars"></i></div>
+    <div class="closeMenuIcon"><i class="fas fa-times hidecross"></i></div>
+<nav class="mobileMenu">
+<ul>
+<li><a href="index.html">Home</a></li>
+<li><a href="index.html">Program</a></li>
+<li><a href="#news">News</a></li>
+</ul>    
+</nav>
+<div class="textIntroAbt">
+    <h2 class="helloText1Abt">"Hello! Hackers in Africa"</h2>
+    <h1 class="hirConferenceText1Abt">HIR AFRICA CONFERENCE 2022</h1>
+    <p class="mainIntro1Abt">The Hackers In Residence Conference is an annual invitation-only
+        gathering of designers, engineers and programmers to discuss the
+        latest developments and innovations in the computer industry. On a
+        daily basis, many hackers only interact virtually, and therefore
+        rarely have face-to-face contact. The conference is a time for
+        hackers to come together to share ideas in Nkwashi city of Zambia.</p>
+    <h3 class="date1Abt">Please get touch if you have questions about the HIR Residence African
+        Conference 2022</h3>
+    <p class="location1Abt">hirafrica@hirnkwashi.org</p>
+</div>
+</header>
+<section class="abtLogoCampaign">
+    <h2 class="conferenceLogoTitle">HIR AFRICA CONFERENCE 2022 LOGO</h2>
+    <div class="logoLine"><hr class="logoHr"/></div>
+    <p class="logoText"> The logo of HIR Africa Conference 2022 was decided through the logo
+        competition last year from 20 May to 28 May.</p>
+        <div><img src="Images/Hirlogo2.jpg" class="logoImage" alt="logo image" ></div>
+        <hr class="sectionDividerLine">
+</section>
+<section class="pastEvents">  
+<h3>See the past HIR Africa Conference</h3>
+<p class="pastText">Take a look at the past HIR Africa conference which took place in
+    Ghana and Rwanda</p>
+    <div class="pastEventImage">
```

Comment:
- [ ] I also noticed the event images were working locally, but not working anymore when deployed let's make sure we fix that.
This is how it looks now: 
![image](https://user-images.githubusercontent.com/87689547/153268352-0d724b5c-ce1f-49a1-8a72-d8ede72cd11a.png)

Reply:
Fixed,thanks